WORKSPACE

```diff
@@ -37,10 +37,10 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/istio/envoy/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
-# Commit date: 2020-08-05 - Branch: release-1.7
-ENVOY_SHA = "a66101fc83a9f246accd9845a840b011e7b0aadc"
+# Commit date: 2020-08-07 - Branch: release-1.7
+ENVOY_SHA = "48a170d0c3e96616a9bce34e4ad2c55fef6936a2"
```

Comment:
It seems there is a newer version. Would you mind to pick it up?

Reply:
Updated. Good catch! I actually updated it before opening the PR, but accidentally made this PR from an old branch.